src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs

```diff
@@ -87,10 +89,10 @@ await retryPolicy.Retry(async () =>
             await _targetHelper.ResetGithubTestEnvironment(githubTargetOrg);
 
             await sourceHelper.CreateBbsProject(bbsProjectKey);
-            await sourceHelper.CreateBbsRepo(bbsProjectKey, "repo-1");
-            await sourceHelper.InitializeBbsRepo(bbsProjectKey, "repo-1");
-            await sourceHelper.CreateBbsRepo(bbsProjectKey, "repo-2");
-            await sourceHelper.InitializeBbsRepo(bbsProjectKey, "repo-2");
+            await sourceHelper.CreateBbsRepo(bbsProjectKey, repo1);
+            await sourceHelper.InitializeBbsRepo(bbsProjectKey, repo1);
+            await sourceHelper.CreateBbsRepo(bbsProjectKey, repo2);
+            await sourceHelper.InitializeBbsRepo(bbsProjectKey, repo2);
```

Comment:
Not sure why this change is necessary, the `repo1` and `repo2` variables are the names of target repos in GitHub the hardcoded ones "repo-1" and "repo-2" are the BBS repo names. By default the target repo name in GitHub for BBS migrations is the combination of the `bbsProjectKey` and the BBS repo name so for example for `repo-1` in BBS the target repo name would be `bbsPorjectKey-repo-1`.

Reply:
These changes have resolved the following error: 
```
FAILURE_REASON: Git source migration failed. Error message: A migration with the same target ssh url has already been queued.
```

Table showing all the CI runs: https://github.ghe.com/github/octoshift/issues/9347#issuecomment-8651851